app/connectors/CodeListConnector.scala

```diff
@@ -59,7 +59,20 @@ class FileBasedCodeListConnector @Inject()(appConfig: AppConfig) extends CodeLis
 
     locals.map { locale =>
       val procedureCodes = codeList.map(factory(_, locale))
-      (locale -> procedureCodes)
+      (locale.getLanguage -> procedureCodes)
+    }.toMap
+  }
+
+  private def readCodesMapFromFile[T <: CommonCode](srcFile: String, factory: (CodeItem, Locale) => T)(implicit locals: Seq[Locale]) = {
```

Comment:
I hope you will consider to replace `readCodesFromFile` and `readCodesMapFromFile` with `loadCodesAsOrderedMap` (in my PR) as it satisfies our requirements, ordering and lookup.

Reply:
Not sure how this is relevant to this PR?